examples/gke/gke_castai_iam/README.md

```diff
@@ -0,0 +1,52 @@
+## Example of creating GCP IAM resources
+
+Following example shows how to create GCP IAM resources required to connect GKE with CAST AI using [castai_gke_iam](https://registry.terraform.io/modules/castai/gke-iam/castai/latest) module.
+
+When creating a service account you can enforce conditional, attribute-based access on `iam.serviceAccountUser` role.
+It can access and act as all other service accounts or be scoped to the ones used by node pools in the GKE cluster.
```

Comment:
Nitpic. Maybe we could add a sentence that it is a "more secure" when service accounts are scoped and it is a best practice to scope them.